src/commands/useReadSmsCommandEventRegistrationParserQuery.js

```diff
@@ -0,0 +1,21 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+
+export const READ_SMS_COMMAND_EVENT_REGISTRATION_PARSER_QUERY = {
+    smsCommand: {
+        resource: 'smsCommands',
+        id: ({ id }) => id,
+        params: {
+            fields: [
+                '*',
+                'program[id,displayName]',
+                'programStage[id,displayName,programStageDataElements[id,dataElement[id,displayName]]',
```

Comment:
perhaps tweak this a bit so you are only getting the fields you need

Reply:
Yes, I'll def. do that once I'll refactor the code